google/cloud/forseti/notifier/notifiers/email_violations.py

```diff
@@ -52,8 +52,12 @@ def __init__(self, resource, inventory_index_id,
                                               global_configs,
                                               notifier_config,
                                               notification_config)
-        self.mail_util = email.EmailUtil(
-            self.notification_config['sendgrid_api_key'])
+        if self.notifier_config.get('email_connector_config'):
+            self.connector = email_factory.EmailFactory(self.notifier_config)\
```

Comment:
Don't use backslash for continuation.  Do something like this.
```
self.connector = (
    email_factory.EmailFactory(self.notifier_config)
    .get_connector())
```

Reply:
Consider it done.

Reply:
Backslash appears when I try to do below as maximum limit of characters per line is exceeded.
self.connector = (
    email_factory.EmailFactory(self.notifier_config)
    .get_connector())
So changed it to below to not use backslash:
            self.connector = email_factory.EmailFactory(
                self.notification_config).get_connector()